netplan_cli/cli/commands/status.py

```diff
@@ -57,20 +65,31 @@ def run(self):
         self.parse_args()
         self.run_command()
 
+    def plain_print(self, *args, **kwargs):
+        if len(args):
+            pattern = r'\[\/?\w+\]'
```

Comment:
thought: I think it's ok in general. But, of course, interfaces with names like `int[external]0` are possible in Linux so it will end up removing part of the name if someone for some reason uses `[]` in interface names out there... but apparently it is possible without this patch as well, as rich will interpret `[]` as formatting.

even funnier, one can affect the rendering by creating an interface called, for example, `[red]`...

Reply:
@daniloegea Thanks, I improved the matching pattern a bit, to match only if there's also a corresponding closing tag. This might have other edge cases (e.g. nested tags...), but maybe those are less common or more under our control. WDYT?

Reply:
nice, it looks like it solves the problem for the plain_print() case.